### PR TITLE
[CI] deselect win32 svm tests causing segmentation faults

### PR DESCRIPTION
**Pull request by icfaust**
### Description 
Remove SVM seg faulting test on windows public CI

Fixes # - _issue number(s) if exists_

- [ ] I have reviewed my changes thoroughly before submitting this pull request.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have updated the documentation to reflect the changes, if necessary (updated in # - _add PR number_)
- [ ] The unit tests pass successfully.
- [ ] I have run it locally and tested the changes extensively.
- [ ] I have resolved any merge conflicts that might occur with the base branch.
- [ ] Git commit message contains an appropriate signed-off-by string _(see [CONTRIBUTING.md](https://github.com/intel/scikit-learn-intelex/blob/main/CONTRIBUTING.md#pull-requests) for details)_
- [ ] I have added a respective label(s) to PR if I have a permission for that.

### CODE REVIEW

**Comment by ethanglaser** [comment]
Looks like this works, but would prefer https://github.com/intel/scikit-learn-intelex/pull/1930

**Comment by samir-nasibli** [comment]
Segfultys happen due to bad input validation. It is expected to be fixed for sklearnex API on #1930 for oneDAL backend lib expected to be addressed as well
